frontend/src/components/dialog/DialogForm.vue

```diff
@@ -71,7 +71,7 @@
               <v-icon v-if="!!submitIcon" left>
                 {{ submitIcon }}
               </v-icon>
-              {{ $tc(submitLabel) }}
+              {{ submitLabel }}
```

Comment:
`submitLabel` still has the default-value `global.button.submit` (similar for cancelLabel, see Lines 99 and 105 below).

I think we need to use a computed for the default-value:
```javascript
submitLabelWithDefaultFallback(){
  return this.submitLabel || this.$tc('global.button.submit')
}
```

and then set the default-value of submitLabel to null.

Reply:
Fixed without a computed in [a8dddbb](https://github.com/ecamp/ecamp3/pull/3128/commits/a8dddbb9981baaf4ad49f5a09cdcbf673c6fa0c0)